### ISSUE TITLE
lr sheduler

### ISSUE THREAD

**Issue by dinarkino** [MisterMap/undeepvo#55]
"The learning rate started from 0.001 and decreased by half for every 1/5 of total iterations"

Добавить возможность выбора через сколько итераций/эпох снижать, либо же что-то умное использовать из торча

**Comment by dinarkino** [comment]
https://pytorch.org/docs/stable/optim.html

```
import torch.optim as optim

SchedulerClass = optim.lr_scheduler.StepLR 
scheduler_params = dict(step_size=3, gamma=0.8)

optimizer_manger = OptimizerManager(lr=lr, 
    scheduler_class=SchedulerClass, scheduler_config=scheduler_params)
```
